tests/test_trainer.py

```diff
@@ -706,6 +707,7 @@ def setUp(self):
         self.n_epochs = args.num_train_epochs
         self.batch_size = args.train_batch_size
 
+    @mark.skip("Skip this test until PT_HPU_LAZY_MODE=0 is set as default for all tests")
```

Comment:
In near future (when we deprecate Lazy mode) it will be set as default in SynapseAI. 

Until then only way to run this test is if we can somehow set this environment variable before any import calls to habana_frameworks, because once habana_framworks is loaded it sets default PT_HPU_LAZY_MODE value (=1) and then changing environment variable with a different value has no impact.  

Since this test runs as a part of full test-suite, 1st test run will load habana_framworks and I do not see a good way of changing environment variable value later.